target/dovecot/90-quota.conf

```diff
@@ -30,7 +30,7 @@ plugin {
 }
 
 service quota-warning {
-    executable = script /usr/local/bin/quota-warning.sh
+    executable = script /usr/local/bin/quota-warning
```

Comment:
I assume because it's placed alongside other shell scripts that omit the extension to parade as binary executables?

Reply:
> I assume because it's placed alongside other shell scripts that omit the extension to parade as binary executables?

Exactly. All files in the `target/bin/` directory lack the `.sh` extension, so I figured, I'd adhere to the already set standard there.

Reply:
> All files in the target/bin/ directory lack the .sh extension,

Ok, depends on the view 😉 I was looking in the container. There, the directory have mixed content (some scripts with extension, some without):

<details>
<summary>Details</summary>

root@mail:/usr/local/bin# ls'

acme_extract
addalias
addmailuser
addrelayhost
addsaslpassword
check-for-changes.sh
check-stack.sh
configomat.sh
daemons-stack.sh
debug-fetchmail
delalias
delmailuser
delquota
excluderelaydomain
fail2ban
fail2ban-wrapper.sh
fetchmailrc_split
fixes-stack.sh
helper-functions.sh
helpers
listalias
listmailuser
misc-stack.sh
open-dkim
postfix-summary
postfix-wrapper.sh
postsrsd-wrapper.sh
print-environment
quota-warning.sh
report-pflogsumm-yesterday
restrict-access
sedfile
setquota
setup
setup-fetchmail
setup-stack.sh
start-mailserver.sh
update-check.sh
updatemailuser
virus-wiper

</details>